.changelog/1820.feature.md

```diff
@@ -0,0 +1 @@
+Add Chinese translation
```

Comment:
filename 1820.feature.md -> 1821.feature.md

Reply:
uuuuh... i have a script to find the next PR number, but github just skipped 1820??